wagtail/images/views/images.py

```diff
@@ -291,8 +292,11 @@ def preview(request, image_id, filter_spec):
     image = get_object_or_404(get_image_model(), id=image_id)
 
     try:
-        response = HttpResponse()
-        image = Filter(spec=filter_spec).run(image, response)
+        # Temporary image needs to be an instance that Willow can run optimizers on
+        temp_image = BytesIO()
+        image = Filter(spec=filter_spec).run(image, temp_image)
+        temp_image.seek(0)
+        response = HttpResponse(temp_image)
```

Comment:
Suggestion: I believe we can use `FileResponse` here for a little optimisation.

Reply:
Yup, works nicely